Mathlib/Logic/Equiv/Array.lean

```diff
@@ -14,8 +14,14 @@ import Mathlib.Logic.Equiv.List
 namespace Equiv
 
 /-- The natural equivalence between arrays and lists. -/
+@[simps! apply symm_apply]
 def arrayEquivList (α : Type*) : Array α ≃ List α :=
-  ⟨Array.toList, Array.mk, fun _ => rfl, fun _ => rfl⟩
+  ⟨Array.toList, List.toArray, fun _ => rfl, fun _ => rfl⟩
+
+/-- `toVector` and `ofVector` together form an equivalence. -/
```

Comment:
As this is in the Equiv namespace, you should probably use the fully qualified names here.